### ISSUE TITLE
consult

### ISSUE THREAD

**Issue by luodi789567890** [Zian-Xu/Swin-MAE#2]
I met some problems  ,could you please tell me your email address? my   email address: ld 2715128882@126.com

**Comment by Zian-Xu** [comment]
Of course, I'm glad to help. My email is 'xuzian1113@foxmail.com'. By the way, my WeChat account is 'xza1610063232'. I'm looking forward to discussing with you!
